src/components/Settings/SettingsAddressbook.vue

```diff
@@ -23,27 +23,25 @@
 <template>
 	<div>
 		<li v-if="editingName" class="new-addressbook">
-			<form id="new-addressbook-form" name="new-addressbook-form"
-				class="new-addressbook__form" addressbooks="[object Object]"
+			<form id="new-addressbook-form" name="new-addressbook-form" class="new-addressbook__form"
 				@submit.prevent="updateAddressbookName">
 				<!-- rename addressbook input -->
 				<input :placeholder="addressbook.displayName"
 					v-model="newName" type="text">
 				<input type="submit" value=""
-					class="new-addressbook__submit inline-button icon-confirm action pull-right">
+					class="new-addressbook__submit icon-confirm">
 			</form>
 		</li>
-		<li v-else :class="{'disabled': !addressbook.enabled}" class="addressbook">
+		<li v-else :class="{disabled: !addressbook.enabled}" class="addressbook">
 			<!-- addressbook name -->
 			<span class="addressbook__name">{{ addressbook.displayName }}</span>
 			<!-- sharing button -->
 			<a href="#" class="addressbook__share icon-shared"
 				@click="toggleShare" />
 			<!-- popovermenu -->
-			<a v-click-outside="closeMenu" href="#" class="addressbook__menu"
-				@click="toggleMenu">
-				<div class="icon-more" />
-				<div :class="{'open': menuOpen}" class="popovermenu">
+			<a v-click-outside="closeMenu" href="#" class="addressbook__menu">
+				<div class="icon-more" @click="toggleMenu" />
```

Comment:
@sleepypioneer this was because the click was triggered on the addressbook__menu as well.
We only want the menu icon to be the toggle here, otherwise we would not have the proper detection since the event would propagate to the parent until stopped by a stopPropagation or similar :)

https://vuejs.org/v2/guide/events.html#Event-Modifiers

Reply:
thank you for the insight 😍